exaudfclient/base/exaudflib/BUILD

```diff
@@ -105,8 +101,6 @@ cc_library(
             "impl/swig/swig_meta_data.h", "impl/swig/swig_meta_data.cc",
             "impl/swig/swig_result_handler.h", "impl/swig/swig_result_handler.cc",
             "impl/swig/swig_general_iterator.h", "impl/swig/swig_general_iterator.cc"],
-    linkstatic = 1,
```

Comment:
add linkstatic = 1, again plus the following description

we enable here linkstatic = 1 such that we can then link the exaudflib statically into the exaudflib_complete.so